### PR TITLE
added translation on reactdomserver page

### PR DESCRIPTION
**Pull request by RiftSplitter**

### CODE REVIEW

File: content/docs/reference-react-dom-server.md [hunk 1]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
 var ReactDOMServer = require('react-dom/server');
 ```
 
-## Overview {#overview}
+## Ikthisar {#overview}
 
-The following methods can be used in both the server and browser environments:
+Metode-metode di bawah ini bisa digunakan di dalam *environment* *server* and *browser* :
```
Comment (spondbob) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Metode-metode di bawah ini bisa digunakan di dalam ruang lingkup *server* dan *browser*:
```

File: content/docs/reference-react-dom-server.md [hunk 2]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
 ### `renderToStaticMarkup()` {#rendertostaticmarkup}
 
 ```javascript
-ReactDOMServer.renderToStaticMarkup(element)
+ReactDOMServer.renderToStaticMarkup(elemen)
 ```
 
-Similar to [`renderToString`](#rendertostring), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
+Serupa dengan [`renderToString`](#rendertostring), kecuali metode ini tidak membuat atribut *DOM* tambahan yang digunakan *React* secara internal, seperti `data-reactroot`. Metode ini bermanfaat jika anda ingin menggunakan *React* untuk menghasilkan halaman statis yang simpel, dikarenakan melepas beberapa atribut tambahan bisa menyimpan beberapa *byte*.
```
Comment (spondbob) [inline content/docs/reference-react-dom-server.md]: Just a couple suggestions:

stripping away > removing > menghapus
save some bytes > menghemat beberapa *byte*

File: content/docs/reference-react-dom-server.md [hunk 3]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
 ### `renderToString()` {#rendertostring}
 
 ```javascript
-ReactDOMServer.renderToString(element)
+ReactDOMServer.renderToString(elemen)
 ```
 
-Render a React element to its initial HTML. React will return an HTML string. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
+Me-*render* sebuah elemen *React* menjadi *HTML* dasarnya. *React* akan mengembalikan sebuah *string* HTML. Anda bisa menggunakan metode ini untuk menghasilkan *HTML* di *server* dan mengirim *markup* sebagai *request* untuk pemuatan halaman yang lebih cepat dan juga memungkinkan mesin pencarian untuk melakukan *crawling* pada halaman-halaman anda demi kepentingan *SEO*.
```
[the remark this answers is not in the document]
Comment (regalius) [reply content/docs/reference-react-dom-server.md]:
```suggestion
Me-*render* sebuah elemen React menjadi HTML biasa. React akan mengembalikan sebuah *string* HTML. Anda bisa menggunakan *method* ini untuk menghasilkan HTML di server dan mengirim *markup* pada permintaan pertama untuk pemuatan halaman yang lebih cepat dan juga memungkinkan mesin pencarian untuk melakukan *crawling* pada halaman-halaman anda demi kepentingan SEO.
```

File: content/docs/reference-react-dom-server.md [hunk 4]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
+Serupa dengan [`renderToString`](#rendertostring), kecuali metode ini tidak membuat atribut *DOM* tambahan yang digunakan *React* secara internal, seperti `data-reactroot`. Metode ini bermanfaat jika anda ingin menggunakan *React* untuk menghasilkan halaman statis yang simpel, dikarenakan melepas beberapa atribut tambahan bisa menyimpan beberapa *byte*.
 
-If you plan to use React on the client to make the markup interactive, do not use this method. Instead, use [`renderToString`](#rendertostring) on the server and [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on the client.
+Jika anda berencana untuk menggunakan *React* di dalam *client* untuk membuat *markup* yang interaktif, anda tidak diperkenankan atau disarankan untuk menggunakan metode ini. Sebaiknya, gunakan metode [`renderToString`](#rendertostring) di dalam *server* dan gunakan metode [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di dalam *client*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Jika Anda berencana untuk menggunakan React di klien untuk membuat *markup* yang interaktif, Anda tidak disarankan untuk menggunakan *method* ini. Sebaiknya, gunakan [`renderToString`](#rendertostring) di server dan gunakan [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di klien.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 5]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
-ReactDOMServer.renderToNodeStream(element)
+ReactDOMServer.renderToNodeStream(elemen)
 ```
 
-Render a React element to its initial HTML. Returns a [Readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) that outputs an HTML string. The HTML output by this stream is exactly equal to what [`ReactDOMServer.renderToString`](#rendertostring) would return. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
+Me-*render* sebuah elemen *React* menjadi *HTML* dasarnya. Mengembalikan sebuah [Readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) yang menghasilkan sebuah *string HTML*. *Output HTML* oleh *stream* ini sama persis dengan hasil atau *output* yang dikembalikan oleh metode [`ReactDOMServer.renderToString`](#rendertostring). Metode ini bisa digunakan untuk menghasilkan *HTML* di *server* dan mengirim *markup* tersebut sebagai *request* untuk pemuatan halaman yang lebih cepat dan juga memungkinkan mesin pencarian untuk melakukan *crawling* pada halaman-halaman anda demi kepentingan *SEO*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Me-*render* sebuah elemen React menjadi HTML biasa. Mengembalikan sebuah [Readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) yang menghasilkan sebuah *string* HTML. Keluaran HTML oleh *stream* ini sama persis dengan keluaran yang dikembalikan oleh [`ReactDOMServer.renderToString`](#rendertostring). Anda dapat menggunakan *method* ini untuk menghasilkan HTML di server dan mengirim *markup* tersebut pada permintaan pertama untuk pemuatan halaman yang lebih cepat dan juga memungkinkan mesin pencarian untuk melakukan *crawling* pada halaman-halaman anda demi kepentingan SEO.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 6]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
-If you call [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on a node that already has this server-rendered markup, React will preserve it and only attach event handlers, allowing you to have a very performant first-load experience.
+Jika anda memanggil [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di dalam sebuah *node* yang sudah memiliki *markup* yang merupakan hasil dari *render* di *server*, *React* akan menyimpan *markup* tersebut dan hanya memasang *event handler* pada *markup* tersebut, memungkinkan anda untuk mendapat performa *first-load experience* yang sangat bagus.
 
-> Note:
+> Catatan:
 >
-> Server-only. This API is not available in the browser.
+> Khusus hanya untuk di *server*. *API* ini tidak bisa digunakan di *browser*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> Hanya untuk server. API ini tidak bisa digunakan di *browser*.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 7]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 >
-> The stream returned from this method will return a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
-
+> *Stream* yang dikembalikan oleh metode ini akan mengembalikan sebuah *byte stream* yang dikodekan dalam *utf-8*. Jika anda membutuhkan *stream* dalam kode selain kode *utf-8*, disarankan anda untuk mencoba melihat projek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang menyediakan *transform streams* untuk *transcoding text*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> *Stream* yang dikembalikan oleh *method* ini akan mengembalikan sebuah *byte stream* yang dienkode dalam utf-8. Jika anda membutuhkan *stream* dalam kode selain utf-8, disarankan untuk mencoba melihat proyek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang menyediakan *transform streams* untuk *transcoding text*.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 8]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
 ```javascript
-ReactDOMServer.renderToStaticNodeStream(element)
+ReactDOMServer.renderToStaticNodeStream(elemen)
 ```
 
-Similar to [`renderToNodeStream`](#rendertonodestream), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
-
-The HTML output by this stream is exactly equal to what [`ReactDOMServer.renderToStaticMarkup`](#rendertostaticmarkup) would return.
+Sama seperti [`renderToNodeStream`](#rendertonodestream), kecuali metode ini tidak membuat atribut *DOM* tambahan yang digunakan *React* secara internal, seperti `data-reactroot`. Metode ini bermanfaat jika anda ingin menggunakan *React* untuk menghasilkan halaman statis yang simple, dikarenakan melepas beberapa atribut tambahan bisa menyimpan beberapa *byte*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Sama seperti [`renderToNodeStream`](#rendertonodestream), bedanya *method* ini tidak membuat atribut DOM tambahan yang digunakan React secara internal, seperti `data-reactroot`. Metode ini bermanfaat jika anda ingin menggunakan React untuk menghasilkan halaman statis yang simple, dikarenakan menghapus beberapa atribut tambahan bisa menghemat beberapa *byte*.
```

File: content/docs/reference-react-dom-server.md [hunk 9]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
+Hasil *output HTML* yang dikembalikan oleh *stream* ini sama persis dengan apa yang dikembalikan oleh [`ReactDOMServer.renderToStaticMarkup`](#rendertostaticmarkup).
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Hasil keluaran HTML yang dikembalikan oleh *stream* ini sama persis dengan yang dikembalikan oleh [`ReactDOMServer.renderToStaticMarkup`](#rendertostaticmarkup).
```

File: content/docs/reference-react-dom-server.md [hunk 10]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
-If you plan to use React on the client to make the markup interactive, do not use this method. Instead, use [`renderToNodeStream`](#rendertonodestream) on the server and [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on the client.
+Jika anda berencana untuk menggunakan *React* di *client* untuk membuat *markup* interaktif, anda tidak diperkenankan atau disarankan untuk menggunakan metode ini. Sebaiknya, gunakan metode [`renderToString`](#rendertostring) di dalam *server* dan gunakan metode [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di dalam *client*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
Jika Anda berencana untuk menggunakan React di klien untuk membuat *markup* yang interaktif, Anda tidak disarankan untuk menggunakan *method* ini. Sebaiknya, gunakan [`renderToString`](#rendertostring) di server dan gunakan [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di klien.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 11]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 
-> Note:
+> Catatan:
 >
-> Server-only. This API is not available in the browser.
+> Khusus hanya untuk di *server*. *API* ini tidak bisa digunakan di *browser*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> Hanya untuk server. API ini tidak bisa digunakan di *browser*.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 12]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 >
-> The stream returned from this method will return a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
+> *Stream* yang dikembalikan oleh metode ini akan mengembalikan sebuah *byte stream* yang dikodekan dalam *utf-8*. Jika anda membutuhkan *stream* dalam kode selain kode *utf-8*, disarankan anda untuk mencoba melihat projek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang menyediakan *transform streams* untuk *transcoding text*.
```
Comment (regalius) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> *Stream* yang dikembalikan oleh *method* ini akan mengembalikan sebuah *byte stream* yang dienkode dalam utf-8. Jika anda membutuhkan *stream* dalam kode selain utf-8, disarankan untuk mencoba melihat proyek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang menyediakan *transform streams* untuk *transcoding text*.
```

What do you think? 😄

File: content/docs/reference-react-dom-server.md [hunk 13]
```diff
@@ -6,7 +6,7 @@ category: Reference
 permalink: docs/react-dom-server.html
 ---
 
-The `ReactDOMServer` object enables you to render components to static markup. Typically, it's used on a Node server:
+Objek `ReactDOMServer` memungkinkan anda untuk me-*render* komponen-komponen ke dalam bentuk *markup* statis. Objek ini digunakan di dalam *server Node* :
```
[the remark this answers is not in the document]
Comment (regalius) [reply content/docs/reference-react-dom-server.md]: All `anda` should be capitalized according to PUEBI 😄

File: content/docs/reference-react-dom-server.md (hunk 1 above)
Comment (spondbob) [reply content/docs/reference-react-dom-server.md]: Don't let the *and* slip away :+1:

**Comment by regalius** [comment]
Hi mas @RiftSplitter , any update regarding this PR? 😄

**Comment by RiftSplitter** [comment]
Hi @regalius sorry for the late reply, I've updated and changed some of the words and capitalization based on suggestions from you and @spondbob. Please review it. :pray:

File: content/docs/reference-react-dom-server.md [hunk 14]
```diff
@@ -15,78 +15,76 @@ import ReactDOMServer from 'react-dom/server';
...
 ```
 
-Similar to [`renderToString`](#rendertostring), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
+Sama seperti [`renderToString`](#rendertostring), bedanya *method* ini tidak membuat atribut DOM tambahan yang digunakan React secara internal, seperti `data-reactroot`. Metode ini bermanfaat jika Anda ingin menggunakan React untuk menghasilkan halaman statis yang simpel, dikarenakan menghapus beberapa atribut tambahan bisa menghemat beberapa *byte*.
 
-If you plan to use React on the client to make the markup interactive, do not use this method. Instead, use [`renderToString`](#rendertostring) on the server and [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on the client.
+Jika Anda berencana untuk menggunakan React di dalam klien untuk membuat *markup* yang interaktif, Anda tidak disarankan untuk menggunakan *method* ini. Sebaiknya, gunakan [`renderToString`](#rendertostring) di dalam server dan gunakan [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) di klien.
```
Comment (rizafahmi) [inline content/docs/reference-react-dom-server.md]: * "di dalam klien" -> "di sisi klien"
* "di dalam server" -> "di sisi server"

Comment (RiftSplitter) [reply content/docs/reference-react-dom-server.md]: Hi @rizafahmi, I've just updated the translation again according to your suggestion. Thanks for the suggestion

**Comment by rizafahmi** [comment]
@regalius all clear!

**Comment by gedeagas** [comment]
Should be good for merge then @rizafahmi @regalius

**Pull request merged**